templates/workers/worker-horizontalpodautoscaler.yaml

```diff
@@ -13,7 +13,7 @@ metadata:
     chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
     heritage: {{ .Release.Service }}
 {{- with .Values.labels }}
-{{ toYaml . | indent 8 }}
+{{ toYaml . | indent 4 }}
```

Comment:
@ianstanton counting these out, I'd expect it to be 6, since we're only doing 2 spaces per indention.